contributing.md

```diff
@@ -19,18 +19,43 @@ your copy.
 Example for .NET:
 
 ```
-$ git clone git@github.com:username/azure-activedirectory-library-for-dotnet.git
-$ rename azure-activedirectory-library-for-dotnet adal.net
+$ git clone git@github.com:username/azure-activedirectory-library-for-dotnet.git adal.net
+$ cd adal.net
 $ git remote add upstream git@github.com:AzureAD/azure-activedirectory-library-for-dotnet.git
 ```
 
+### Initial build of ADAL.Net
+We recommand that you use Visual Studio 2017 with Xamarin, .Net Core and UWP installed. 
+You will also need Visual Studio 2015 update 3 with the windows 8.1 SDK installed in order to build ADAL.NET for the WinRT platform
 > Important node: 
 ADAL.Net uses a multi-targeting projects to generate assemblies for several platforms. There are issues with the Nuget package manager in Visual Studio 2017, not recogizing nuget pacakges in the ADAL.NET project. Therefore you'd want to restore the nuget packages using this command line:
 
 ```
 $ msbuild ADAL.NET.NoWinRT.sln /t:restore 
 ```
 
+you can then either build ADAL.Net in Visual Studio or from the command line:
+```
+$ msbuild ADAL.NET.NoWinRT.sln /t:build /p:configuration=release 
+```
+
+To run the unit tests:
+-Skip strong name Verification for this assembly for on your machine (run in command prompt as an admin)
+```
+sn -Vr *,31bf3856ad364e35
+```
+
+Run the unit tests:
+```
+vstest.console tests\Test.ADAL.NET.Unit\bin\release\net462\Test.ADAL.NET.Unit.dll
```

Comment:
you also need to run integration test dll.

Reply:
addressed